skimage/segmentation/_felzenszwalb_cy.pyx

```diff
@@ -12,50 +12,56 @@ from ..measure._ccomp cimport find_root, join_trees
 from ..util import img_as_float
 
 
-def _felzenszwalb_grey(image, double scale=1, sigma=0.8,
```

Comment:
It might be better to rename it to something like `_felzenszwalb_multichannel` or just `_felzenszwalb_cy`.